docs/examples/Titanic-Binary-Classification.md

```diff
@@ -36,13 +36,13 @@ In the main function, we create a spark session as per usual:
 import org.apache.spark.SparkConf
 import org.apache.spark.sql.SparkSession
 
-val conf = new SparkConf().setAppName("TitanicPrediction")
-implicit val spark = SparkSession.builder.config(conf).getOrCreate()
+implicit val spark = SparkSession.builder.config(new SparkConf()).getOrCreate()
+import spark.implicits._ // Needed for Encoders for the Passenger case class
 ```
 
-We then define the set of raw features that we would like to extract from the data. The raw features are defined using [FeatureBuilders](/Developer-Guide#featurebuilders), and are strongly typed. TransmogrifAI supports the following basic feature types: Text, Numeric, Vector, List , Set, Map. In addition it supports many specific feature types which extend these base types: Email extends Text; Integral, Real and Binary extend Numeric; Currency and Percentage extend Real. For a complete view of the types supported see the [Type Hierarchy and Automatic Feature Engineering](/Developer-Guide#type-hierarchy-and-automatic-feature-engineering) section in the Documentation.
+We then define the set of raw features that we would like to extract from the data. The raw features are defined using [FeatureBuilders](../developer-guide#featurebuilders), and are strongly typed. TransmogrifAI supports the following basic feature types: Text, Numeric, Vector, List , Set, Map. In addition it supports many specific feature types which extend these base types: Email extends Text; Integral, Real and Binary extend Numeric; Currency and Percentage extend Real. For a complete view of the types supported see the [Type Hierarchy and Automatic Feature Engineering](../developer-guide#type-hierarchy-and-automatic-feature-engineering) section in the Documentation.
```

Comment:
I dont think `../` prefixes would work once we deploy to https://docs.transmogrif.ai
were the links broken? how did you test them?

Reply:
The links are broken on the website, but only for that paragraph. I just imitated the style of the links that work in the other paragraphs. How do I link a page that will work when deployed?

Reply:
Try running the docs server locally as described here - https://github.com/salesforce/TransmogrifAI/tree/master/docs#docs

Reply:
Oh, you're right. We actually have them as `../` elsewhere.

Reply:
They work.